### PR TITLE
Vue is working, grid and tiles started

### PR DESCRIPTION
**Pull request by jason00111**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    ["es2015", { "modules": false }]
+  ]
```
Comment (devonwesley) [inline .babelrc:4]: Whats does this do? ``` {"module": false}

File: package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "my-project",
+  "description": "A Vue.js project",
+  "version": "1.0.0",
+  "author": "jason00111 <emberleyjason@gmail.com>",
+  "private": true,
+  "scripts": {
+    "dev": "cross-env NODE_ENV=development webpack-dev-server --open --inline --hot",
+    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules"
```
Comment (devonwesley) [inline package.json:9]: What is ```cross-env``` doing?

File: src/main.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import Vue from 'vue'
+import App from './App.vue'
+
+new Vue({
+  el: '#app',
+  render: h => h(App)
+})
```
Comment (devonwesley) [inline src/main.js:7]: What is the function ```h(App)``` doing?

**Comment by devonwesley** [comment]
I get what's going on. Go ahead and merge. I just left a couple comments out of curiosity. If you can reply please. I would like to learn with y'all.

File: .babelrc (hunk 1 above)
Comment (imsolost) [reply .babelrc:4]: It is necessary for tree shaking, webpack 2 can handle ES6 modules syntax. Tree shaking eliminates unused exports.

File: package.json (hunk 2 above)
Comment (imsolost) [reply package.json:9]: This micro-lib allows you to provide a script which sets an environment using unix style and have it work on windows too

File: src/main.js (hunk 3 above)
Comment (imsolost) [reply src/main.js:7]: Aliasing createElement to h is a common convention you’ll see in the Vue ecosystem and is actually required for JSX. If h is not available in the scope, your app will throw an error.

**Pull request merged**